src/language-js/parse/babel.js

```diff
@@ -107,6 +107,13 @@ function createParse(parseMethod, ...optionsCombinations) {
       }));
     }
 
+    if (text.includes("#{") || text.includes("#[")) {
+      combinations = combinations.map((options) => ({
+        ...options,
+        plugins: [...options.plugins, recordAndTuplePlugin],
+      }));
```

Comment:
Let's use `appendPlugins(options, [recordAndTuplePlugin])`, as it's more clear,  and same for the pipeline one below. It should work

Reply:
Ah, `appendPlugins` don't accept `options`...

Reply:
What do you think, change `appendPlugins` to 

```js
const appendPlugins = (plugins, options = parseOptions) => ({
  ...options ,
  plugins: [...options .plugins, ...plugins],
});
```

then use it?

Reply:
976be28